docusaurus/docs/user-docs/settings/review-workflows.md

```diff
@@ -25,6 +25,10 @@ The default workflow is configured to have 4 stages: To do, In progress, Ready t
 
 Before being available in the [Content Manager](/user-docs/content-manager/reviewing-content), review workflows must be configured from ![Settings icon](/img/assets/icons/settings.svg) *Settings > Global settings > Review Workflows*. The Review workflows settings are only available to users with the Super Admin role by default. Other roles must be granted the **Review workflows** permissions. See [Users, Roles, & Permissions](/user-docs/users-roles-permissions) for more information.
 
+:::note
+Currently all roles using Review Workflows in the content-manager need to have at least "Read" permission for workflow settings, defined in *Settings > Global settings > Roles > Edit a role > Settings > Review Workflows*.
```

Comment:
```suggestion
Currently all roles using Review Workflows in the content-manager need to have at least "Read" permission for workflow settings, defined in *Settings > Administration Panel > Roles > Edit a role > Settings > Review Workflows*.
```

Thanks, Gustav!

Are we talking about the following permissions?
![Screenshot 2023-07-31 at 15 03 03](https://github.com/strapi/documentation/assets/4233866/68df1add-a467-4a06-a16b-50de0434c363)

If yes, we should update the text as suggested 🙂

Reply:
Sounds good to me. I've used it because that is the way it was written in the previous paragraph. Should we update both?